extensions/compose/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "build": "vite build && node ./scripts/build.js",
+    "test": "npm run build && vitest run -r src --coverage",
```

Comment:
I would only do the vitest part, not the build

People can do `npm build` &&  `npm test`